.gitignore

```diff
@@ -11,4 +11,5 @@ x64/
 build/
 bld/
 [Bb]in/
-[Oo]bj/
\ No newline at end of file
+[Oo]bj/
+InfernalRobotics/InfernalRobotics/InfernalRobotics.csproj
```

Comment:
Oh, I accidentally included .gitignore in commit. What is the easiest way to fix that?